src/modes/map/map_treasure.cpp

```diff
@@ -319,14 +319,14 @@ void TreasureSupervisor::_UpdateList()
             _selection_name.SetText(UTranslate("Drunes"));
             _selection_icon = GlobalManager->Media().GetDrunesIcon();
 
-            // Plural gettext for the 1st sentence.
             const std::string temp_string =
+                // Part 2 of "With the additional %u drune(s) found in this treasure added, ... "
```

Comment:
About that commit, I would state the copy/paste sentence in comment, but rather what it does. Otherwise, whenever we change the sentence, we have not to forget to change the comment as well, which won't happen. ;)
Please change this.